packages/cli/src/commands/apps/unlock.ts

```diff
@@ -0,0 +1,32 @@
+import color from '@heroku-cli/color'
+import {Command, flags} from '@heroku-cli/command'
+import {ux} from '@oclif/core'
+import * as Heroku from '@heroku-cli/schema'
+
+export default class Unlock extends Command {
+    static topic = 'apps'
+    static description = 'unlock an app so any team member can join'
+    static aliases = ['unlock']
+    static flags = {
+      app: flags.app({required: true}),
+      remote: flags.remote(),
+    }
+
+    public async run(): Promise<void> {
+      const {flags} = await this.parse(Unlock)
+      const {app} = flags
+      const {body: appResponse} = await this.heroku.get<Heroku.TeamApp>(`/teams/apps/${app}`)
+      const appName = appResponse.name ?? app
+      if (!appResponse.locked) {
+        throw new Error(`Error: cannot unlock ${color.cyan(appName)}\nThis app is not locked.`)
```

Comment:
I would use

```suggestion
        ux.error(`cannot unlock ${color.cyan(appName)}\nThis app is not locked.`, {exit: 1})
```
just for consistency with usage throughout the rest of the code.

At least the `Error: ` at the start of the message should be removed because it is added automatically by Oclif error handler and it gets duplicated in the console (i.e. `Error: Error: cannot unlock...`.